src/main/resources/db/migration/V202408091410__owner_table.sql

```diff
@@ -0,0 +1,8 @@
+create table owner(
+    id serial primary key ,
+    name varchar(100) not null ,
+    company_id int not null,
+    employee_id varchar(100) not null
+);
+
+CREATE INDEX idx_owner_id_employee_id ON owner(id, employee_id);
```

Comment:
There is no point in putting an index on a primary key
it should be on company_id and employee_id

Reply:
Oops...
Got it!